tvtk/wrapper_gen.py

```diff
@@ -105,13 +105,14 @@ def get_trait_def(value, **kwargs):
             dtype = 'int64'
         elif dtype == 'long':
             dtype = 'int64'
+        dtype = '"{}"'.format(dtype) if dtype is not None else 'None'
```

Comment:
Take care of "cannot convert 'None' to dtype" trait errors. Directly pass None instead of 'None' string